css/src/nc12.scss

```diff
@@ -72,3 +72,205 @@
 	box-shadow: unset;
 	filter: drop-shadow(0 1px 3px rgba(179, 179, 179, .75));
 }
+
+$color-main-text: #000;
```

Comment:
This is not needed, nc12 has these variables as well.
https://github.com/nextcloud/server/blob/stable12/core/css/variables.scss

Reply:
Thanks, I will remove it.